libs/labelbox/tests/data/annotation_import/conftest.py

```diff
@@ -194,6 +193,21 @@ def llm_human_preference_data_row(global_key):
     return llm_human_preference_data_row
 
 
+@pytest.fixture(scope="module")
```

Comment:
```suggestion
@pytest.fixture(scope="session")
```

Reply:
Could do that, but then we could also do it for all the other similar fixtures. What do you think?